__tests__/app.test.js

```diff
@@ -168,3 +168,100 @@ describe("GET", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  it("should respond with username and body with comment", () => {
+    const commentInput = {
+      body: "I am slow but i am committed",
+      author: "mallionaire",
+    };
+    return request(app)
+      .post("/api/reviews/4/comments")
+      .send(commentInput)
+      .expect(201)
+      .then((response) => {
+        const { body } = response;
+        const { comment } = body;
+
+        expect(comment).toEqual({
+          comment_id: expect.any(Number),
+          body: "I am slow but i am committed",
+          votes: 0,
+          author: "mallionaire",
+          review_id: 4,
+          created_at: expect.any(String),
+        });
+      });
+  });
+  it("should respond with username and body with comment", () => {
```

Comment:
Just make sure to change this test description to match what the test is asserting against